cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -206,7 +215,7 @@ private StructuredRow resultSetToRow(ResultSet resultSet) throws SQLException {
     // We don't close the statement here because once it is closed, the result set is also closed.
     try {
       PreparedStatement statement = connection.prepareStatement(scanQuery);
-      statement.setFetchSize(SCAN_FETCH_SIZE);
+      statement.setFetchSize(cConf.getInt(SCAN_FETCH_SIZE));
```

Comment:
Major: getInt with a default in case the config is not specified in cdap-site.

Reply:
I don't think it's needed as default is specified in `cdap-default.xml`. It's better not to have defaults in two places as it may get out of sync

Reply:
Agree with @tivv .